src/Compute/Compute/help/New-AzVM.md

```diff
@@ -1154,7 +1154,8 @@ Accept wildcard characters: False
 ```
 
 ### -Zone
-Specifies the zone list of the virtual machine.
+Specifies the zone of the virtual machine. Although it takes in an array of zones, virtual machines do not support multiple availability zones.
+The allowed value depend on the capabilities of the region. Allowed value will normally be 1,2, or 3. More information on [Azure availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview#availability-zones).
```

Comment:
I would go with this:

Specifies the availability zone of the virtual machine. 

```console
The allowed value depend on the capabilities of the region. Allowed value will normally be 1, 2, or 3. 

> Although it takes in an array of zones, virtual machines do not support multiple availability zones.
  More information on [Azure availability zones](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-overview#availability-zones).
```

which uses the ">" for the note and does not distract the user from the actual juicy part of the description.

Reply:
leaving it as is. thank you for your input.